src/components/DatePicker/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/date-picker",
-  "version": "1.1.9",
+  "version": "1.1.10",
```

Comment:
you don't need to update the version num here.  it will be done as part of lerna process detailed in readme

Reply:
Severin mentioned this already.  resolving.